packages/parser/README.md

```diff
@@ -16,6 +16,8 @@
 npm install @typescript-eslint/parser --save-dev
 ```
 
+Note: `@typescript-eslint/parser` assumes you have `typescript` already installed. Please read the documentation [here](https://github.com/typescript-eslint/typescript-eslint#supported-typescript-version) for more details.
```

Comment:
probably want this in `eslint-plugin` as well, as `eslint-plugin` also directly consumes typescript.